graphics/poppler/Portfile

```diff
@@ -125,6 +126,14 @@ if {${subport} ne ${name}} {
     conflicts_build ${name}
 }
 
+# build pdfsig
+variant pdfsig description {Build pdfsig binary} {
+	configure.args-delete -DWITH_NSS3=OFF
+	configure.args-append -DWITH_NSS3=ON
+	depends_build-append port:nss
```

Comment:
This line can be removed. The `depends_lib` dependency on it already covers this.